lib/models/package-info-cache.js

```diff
@@ -0,0 +1,1049 @@
+'use strict';
```

Comment:
I guess we could, though it would then sort of pollute the directory with what really are internal items in the cache (the PackageInfo, the NodeModuleList, the ErrorList and ErrorEntry objects). The cache is sort of irrelevant without them. I guess we could make a subdirectory of 'lib/models', but didn't know what you guys preferred.

Reply:
@twokul, any thoughts on my comment? I've tried to add more documentation comments (header for each class, for example), but I agree that it's not a small file.

Reply:
My recommendation would be a `package-info` folder, containing:

* `index.js` the package-info itself
* each atomic unit as a sibling file

Reply:
Done and checked in. The directory is called package-info-cache, and each class is its own file.